Apps/JobScheduler/src/JobScheduler.csproj

```diff
@@ -47,6 +47,7 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="appsettings.json" CopyToPublishDirectory="Always" />
+    <None Include="**/*" />
```

Comment:
May Including everything here cause some side effects? @sslaws, Do you mind to explain why do we need to add this line?

Reply:
We don't, Visual Studio must have done this.  I've removed it.